pkgs/applications/terminal-emulators/iterm2/default.nix

```diff
@@ -1,48 +1,35 @@
-{ lib, stdenvNoCC, fetchFromGitHub }:
+{ lib, pkgs, stdenv }:
 
  /*
- This derivation is impure: it relies on an Xcode toolchain being installed
- and available in the expected place. The values of sandboxProfile
- are copied pretty directly from the MacVim derivation, which
- is also impure. In order to build you at least need the `sandbox`
- option set to `relaxed` or `false`.
+ This cannot be built from source as it requires entitlements and
+ for that it needs to be code signed. Automatic updates will have
+ to be disabled via preferences instead of at build time.
  */
 
```

Comment:
```suggestion
let
  appName = "iTerm.app";
in
```

Reply:
Used in the `installPhase`. I prefer to use let statements as this allows me to get rid of recursive derivations. E.g if you add the `version` to this let statement and apply the suggestion from @SuperSandro2000 you could get rid of the recursive derivation. But this is more a personal preference.

Reply:
It makes only a few bytes difference removing rec and replacing it with let in and there is no need here.